asv/benchmarks/setup/bench_model.py

```
@@ -46,6 +50,11 @@ def time_initialize_model(self, robot, num_envs):
         _model = builder.finalize()
         wp.synchronize_device()
 
+    @skip_benchmark_if(wp.get_cuda_device_count() == 0)
+    def time_initialize_solver(self, robot, num_envs):
```

Ruff (0.12.2)

Unused method argument: `num_envs`

(ARG002)